### PR TITLE
Add Akkoma icon

### PR DESCRIPTION
**Pull request by Sevichecc**
[Akkoma](https://akkoma.social/) is a new fediverse social software based on Pleroma. An increasing number of instances are using Akkoma. Here are some data on Akkoma: [Fediverse Observer](https://akkoma.fediverse.observer/stats). Currently it has 10,526 active montly users.

image

**Similarweb rank:** 8,945,151
### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

Since Akkoma has not provided official branding materials, I picked the hex color from their favicon: https://akkoma.dev/AkkomaGang/akkoma/src/branch/develop/docs/docs/images/favicon.png

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -403,6 +403,11 @@
             "hex": "0096D6",
             "source": "https://www.akamai.com"
         },
+        {
+            "title": "Akkoma",
+            "hex": "462D7B",
+            "source": "https://akkoma.dev/assets/img/logo.svg"
```
Comment (adamrusted) [inline _data/simple-icons.json:409]:
```suggestion
            "source": "https://akkoma.dev"
```

We can shorten this down to just the homepage, given the SVG is embedded there.

**Review by adamrusted — CHANGES_REQUESTED** [review]
Slight nit-pick on the source URL. It would also appear that neither website currently meets our inclusion criteria. Have we included other fediverse instances with similar performance?

**Comment by service-paradis** [comment]
I'm not sure it meets our popularity requirements. Do you have more information that we can use?

**Comment by service-paradis** [comment]
Closing since it is out of scope.

**Pull request closed without merging**